internal/daemon/pipeline/pipeline.go

```diff
@@ -77,6 +77,9 @@ func processNestedKeys(pipelines Pipelines) {
 func (a Archive) Validate(addMissingID bool) error {
 	var inputs, outputs int
 	for _, pipeline := range a.Pipelines {
+		if strings.HasSuffix(pipeline.Config, "/") {
+			pipeline.Config += "*"
```

Comment:
I thought by default logstash picked up just *.conf files and not every file?

https://www.elastic.co/guide/en/logstash/master/dir-layout.html suggests that `/etc/logstash/conf.d/*.conf` is the default for standard pipelines.

Can try and verify this tomorrow for multiple pipelines.

I can try to verify this tomorrow

Reply:
Had a quick chance to check and looks like I'm wrong.  It does indeed use all files underneath the directory.  Looks like it only applies to standard single pipelines.